### PR TITLE
stat collector testing

### PR DESCRIPTION
**Pull request by parmjeetsingharora**

### CODE REVIEW

File: StatCollectorTesting.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 public class StatCollectorTesting {
 
   public static void main(String[] args) {
-    System.out.println("Test");
+    System.out.println("Test 2");
```
Comment (parmjeetsingharora) [inline StatCollectorTesting.java:4]: test

File: StatCollectorTesting.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+    System.out.println("Test 3");
```
Comment (param2143) [inline StatCollectorTesting.java:5]: bug

**Comment by param2143** [comment]
test

**Comment by parmjeetsingharora** [comment]
run test

File: StatCollectorTesting.java (hunk 2 above)
Comment (parmjeetsingharora) [reply StatCollectorTesting.java:5]: solved

File: StatCollectorTesting.java (hunk 1 above)
Comment (parmjeetsingharora) [reply StatCollectorTesting.java:4]: testing

File: StatCollectorTesting.java (hunk 2 above)
Comment (param2143) [reply StatCollectorTesting.java:5]: again test

Comment (parmjeetsingharora) [reply StatCollectorTesting.java:5]: reply

File: StatCollectorTesting.java [hunk 3]
```diff
@@ -1,6 +1,7 @@
 public class StatCollectorTesting {
```
Comment (param2143) [inline StatCollectorTesting.java:1]: name

**Review by param2143 — CHANGES_REQUESTED** [review]
resolv ecommet

**Pull request merged**